nx-X11/include/extensions/Imakefile

```diff
@@ -42,15 +42,6 @@ XINERAMAHEADERS = Xinerama.h panoramiXext.h panoramiXproto.h
 #if BuildRender || BuildXftLibrary
 RENDERHEADERS = render.h renderproto.h
 #endif
-#if BuildXfixes || BuildXfixesLibrary
-XFIXESHEADERS = xfixeswire.h xfixesproto.h
-#endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
-#endif
-#if BuildComposite || BuildCompositeLibrary
-COMPOSITEHEADERS = composite.h compositeproto.h
-#endif
```

Comment:
Ok. Interesting. This may become tricky here then.

To be clear on this, if we use e.g. libxcompose1 instead of shipping outdate libnx-xcompose1, means we drop composite.h and compositeproto.h from nx-X11/include/extensions.

Does it also mean, that we update nx-X11/programs/Xserver/composite and take some latest X.org version of that folder and replace the one in nx-libs with that?

I have already played with that, but it will surely end up in replacing the whole nx-X11/programs/Xserver/ code tree, because of the manifold ABI changes inside xorg-xserver over the past ten years.